core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -139,11 +140,21 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
    * we need to add a delay of X to W such that O * W / (W + X) = T.
    * Solving for X, we get X = (O - T)/T * W.
```

Comment:
Per the comment in the RB - can we document the discussion on the right vs conservative formula here?